tunelo/flask.py

```diff
@@ -26,6 +26,7 @@ def create_app(test_config=None):
     else:
         # TODO: set anything relevant from CONF
         app.config.update(PROPAGATE_EXCEPTIONS=True)
+        app.config.update(JSON_SORT_KEYS=False)
```

Comment:
?

Reply:
This was done so that the returned object from the API would match the order of the design, rather than sorted alphabetically.